src/platform/user/profile/vet360/actions/transactions.js

```diff
@@ -247,7 +246,6 @@ export const validateAddress = (
       addressValidationType: fieldName,
       addressValidationError: true,
       addressFromUser: { ...payload },
-      validationKey: null, // add this in when changes are made to API / override logic
```

Comment:
Per Lihan, we will not have an instance where the validationKey is provided with non 200 response.  Removed this and associated with copy since it can never be reached